---
layout: single
author_profile: true
sidebar:
    nav: research
---

## Methods for determining nucleic acid dynamic ensembles at atomic resolution: role in molecular recognition and RNA folding  {#nucleic-acid-ensembles}

<div markdown="1" style="font-size: 18px;">
While it is now trivial to physically see cellular dynamics, such as a cell dividing, with a microscope, the technology does not yet exist to visualize the structural or chemical dynamics of biochemical reactions on an atomic level, even for the simplest molecules. We are developing hybrid experimental-computational methods that aim to accomplish this goal and determine dynamic-ensembles of nucleic acids at atomic resolution. Our effort thus far has led to the first 3D experimental visualization of motions in RNA at atomic resolution. We continue to improve the ensemble determination approaches as well as building frameworks for utilizing these dynamic ensemble descriptions to elucidate the mechanisms of RNA adaptive recognition; sequence-specific DNA-protein recognition; and RNA tertiary folding. See this Movie of an unbound HIV-1 RNA dynamically sampling its many different drug-bound conformations. 
{: .text-justify}
</div>

#### Representative Publications:

<div markdown="1" style="font-size: 16px;">
- Zhang Q, Sun X, Watt EW, and Al-Hashimi HM (2006) Resolving the Motional Modes that Code for RNA Adaptation. Science 311: 653-656
- Zhang Q, Stelzer AC, Fisher CK, and Al-Hashimi HM (2007) Visualizing Spatially Correlated Dynamics that Directs RNA Conformational Transitions. Nature 450:1263-1267
- Bailor M, Sun X, and Al-Hashimi HM (2010) Topology Links RNA Secondary Structure with Global Conformation Dynamics and Adaptation. Science 327(5962):202-206
</div>


## Fleeting conformational states of nucleic acids: role in spontaneous mutations, RNA folding, and mechanisms of epitranscriptomics {#excited-states}

<div markdown="1" style="font-size: 18px;">
In biology, some of the most functionally important conformational states of biomolecules exist in exceptionally low abundance (<0.01%); last for as little as a millionth of a second; and can involve subtle sub-angstrom movements of atoms. Our group has advanced approaches that combine NMR spectroscopic techniques with chemical traps for capturing such fleeting states in nucleic acids. This led to the discovery of rare nucleic acid conformational states that are implicated the mechanisms of spontaneous mutations; translational errors; viral replication; and RNA tertiary folding. We are examining the function of these fleeting states as well as their potential as opportune drug targets by using mutations to knock out or trap these states in in vitro and cell-based experiments. We are also investigating how epitranscriptomic modifications may exert their biological function by modulating the dynamics to these fleeting conformational states. See this Movie for an example of fleeting base pair states that are thought to underlie spontaneous mutations. 
{: .text-justify}
</div>

#### Representative Publications:

<div markdown="1" style="font-size: 16px;">
- Xue Y, Gracia B, Herschlag D, Russell R, and Al-Hashimi HM (2016) Visualizing Formation of an RNA Folding Intermediate through a Fast Highly Modular Secondary Structure Switch. Nature Communications 7:11768
- Kimsey I, Petzold K, Sathyamoorthy B, Stein Z, and Al-Hashimi HM (2015) Visualizing Transient Watson-Crick Like Mispairs in DNA and RNA Duplexes. Nature 519(7543):315-320
- Dethoff L, Petzold K, Chugh J, Casiano-Negroni A, and Al-Hashimi HM (2012) Visualizing Transient Low-Populated Structures of RNA. Nature 491(7426):724-728
</div>

## Hoogsteen Base Pairs: redefining the DNA double helix {#hoogsteen-base-pairs}

<div markdown="1" style="font-size: 18px;">
Our group made the discovery that in canonical DNA duplexes, Watson-Crick base pairs exist in dynamic equilibrium with Hoogsteen base pairs in which the guanine or adenine base flips ~180 degrees to form a unique set of hydrogen bonds with its cytosine or thymine base complement. We are developing and applying methods based on solid state NMR and infrared (IR) spectroscopy to resolve Watson-Crick and Hoogsteen base pairs and their dynamic equilibria in nucleosomes and chromatin. We are examining the role of Hoogsteen base pairs in exposing the Watson-Crick face of nucleotide bases to mutagenic damage. We are also developing a new sequencing approach (‘Hoog-seq’) to map Hoogsteen base pairs genome-wide in vivo. We are interested in examining the enrichment of Hoogsteen base pairs relative to nucleosome positioning and other genomic features to assess potentially broader roles for Hoogsteen base pairs in gene expression and genome stability.  See this Movie to see the transitions between Watson-Crick and Hoogsteen base pairs. 
{: .text-justify}
</div>

#### Representative Publications:

<div markdown="1" style="font-size: 16px;">
- Alvey HS, Gottardo FL, Nikolova EN, and Al-Hashimi HM (2014) Widespread Transient Hoogsteen Base-Pairs in Canonical Duplex DNA with Variable Energetics. Nature Communications 5:4786
- Nikolova E, Kim E, Wise A, O’Brien P, Andricioaei I, and Al-Hashimi HM (2011) Transient Hoogsteen Base-pairs in Canonical Duplex DNA. Nature 470(7335):498-502
</div>

## RNA and DNA-targeted drug discovery {#drug-discovery}

<div markdown="1" style="font-size: 18px;">
We are developing a dynamic-ensemble based virtual screening platform for enabling the rational discovery of nucleic acid targeting small molecule therapeutics. Our effort has so far focused on targeting functionally important non-coding RNA in HIV-1 as well as on the development of anti-cancer therapies targeting duplex DNA. By computationally docking a dynamic ensemble of RNA structures determined using our NMR-based methods, rather than a single static structure, we have overcome a major obstacle in structure-based RNA-targeted drug discovery. Docking to an HIV-1 RNA target ensemble resulted in the first computational discovery of an RNA-targeting small molecule with in vivo activity. We are also extending these approaches to target rare and unusual conformational states of nucleic acids that exist in low abundance (<1%) and for short periods of time (lifetimes on the order of milliseconds). These projects marry NMR spectroscopic, computational, in vitro and cell-based studies.  See this Movie for example of RNA ensemble used in computational docking.
{: .text-justify}
</div>

#### Representative Publications:

<div markdown="1" style="font-size: 16px;">
- Stelzer AC, Frank AT, Kratz JD, Swanson M, Gonzalez-Hernandez MJ, Andricioaei I, Markovitz DM, and Al-Hashimi HM (2011) Discovery of HIV-1 Inhibitors by Targeting an RNA Dynamic Ensemble. Nature Chemical Biology 7(8):553-559
</div>

